internal/split/init.go

```diff
@@ -0,0 +1,29 @@
+package split
+
+import (
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/newrelic/newrelic-cli/internal/config"
+	configAPI "github.com/newrelic/newrelic-cli/internal/config/api"
+)
+
+var (
+	SplitService *Service
+)
+
+func Init() {
+	region := configAPI.GetConfigString(config.Region)
+	apiKey := GetAPIKeyByRegion(strings.ToLower(region))
+
+	service, err := NewSplitService(apiKey, region)
+	if err != nil {
+		log.Errorf("could not initialize SplitService: %s\n", err)
```

Comment:
This should be `log.Fatalf`, or the method should return an error, unless we are committing to checking for nil during every usage of `SplitService`.  Currently if there is an initialization error, we will experience a panic when using the instance.